src/classes/OPP_StageMappingUtil.cls

```diff
@@ -93,8 +96,118 @@ public with sharing class OPP_StageMappingUtil {
     @testVisible private static void initializeStates() {
         // These are temporary values to be used only during interim for internal development on Abacus project.
         // This will be updated in March 2019 timeframe with revised code to retrieve from Custom Metadata instead
-        COMMITTED_STATE_SET = new Set<String> {'closed won'};
-        FINALIZED_STATE_SET = new Set<String> {'finalized'};
+        // COMMITTED_STATE_SET = new Set<String> {'closed won'};
+        // FINALIZED_STATE_SET = new Set<String> {'finalized'};
+
+        // If either the committed state set or the finalized state set is null, query for everything and 
+        // reinitialize everything. 
+        if (COMMITTED_STATE_SET == null || FINALIZED_STATE_SET == null) {
+            COMMITTED_STATE_SET = new Set<String>();
```

Comment:
If you switch to the approach I outlined, you can eliminate initialization here for the two sets.